app/src/main/res/values/strings.xml

```diff
@@ -107,4 +107,6 @@
     <string name="details">Details</string>
     <string name="tasks">Tasks</string>
     <string name="add_new_task">Add a new task</string>
+    <string name="confirm_logout">Confirm Logout</string>
```

Comment:
Change this to "Yes"

Reply:
@m-murad this string is used as a title for the dialog. You want the title of the dialog to be "Yes"?

Reply:
```suggestion
    <string name="confirm_logout">Confirm logout</string>
```

maybe like this @jashpreet

Reply:
Sure !